07LessonSeven/homework/css/style.css

```diff
@@ -1,59 +1,82 @@
 body {
-  font-family: sans-serif;
-  font-weight: lighter;
+  font-family: sans-serif, Arial;
+  font-size: 14px;
 }
+
+#container {
+  display: block;
+  width: 100%;
+  height: 100%;
+  margin: 0 auto;
+}
+
 h1 {
-  padding-left: 170px;
+  padding-left: 12.5%;
 }
+
 nav {
-  background-color: #282d2e;
-  height: 80px;
+  height: 60px;
+  padding-top: 16px;
+  margin-bottom: 10px;
 }
-a:link {
-  text-decoration: none;
-  color: white;
+
+nav, #looking {
+  background-color: #313131;
 }
+
 ul {
   list-style-type: none;
-  padding-left: 170px;
-
+  padding-left: 12.5%;
 }
+
 li {
-  float: left;
-  font-size: 16px;
-  padding-right: 25px;
-  padding-top: 28px;
-}
-div {
-  display: flex;
```

Comment:
Thanks for using flex. It just makes your life as a web designer much easier. Floats are important to understand but use only when truly necessary. Keep taking advantage of the modern properties as they become available, like flexbox. :smiley: